client/src/components/SubstraitGraph.vue

```diff
@@ -85,6 +99,24 @@ export default {
       };
       image.src = svgUrl;
     },
+    async generateLink() {
+      try {
+        const resp = await axios.post("/api/save/", {
+          json_string: store.plan,
+          validation_levels: store.validation_override_levels,
+        });
+
+        const textField = document.createElement("textarea");
+        textField.innerText = this.currentUrl + "/plan/" + resp.data;
+        document.body.appendChild(textField);
+        textField.select();
+        document.execCommand("copy");
+        textField.remove();
+        alert("Link copied to clipboard!");
```

Comment:
Alert works for now but we will want something we can style better eventually.  Does bootstrap have any kind of dialog or alerting component?

Reply:
Yes, we can have a dialog component from Bootstrap. I will implement that in a follow-up PR.